packages/datahub-data/src/object-list.ts

```diff
@@ -0,0 +1,67 @@
+'use server';
+
+import {objectLists, insertObjectItemSchema, objectItems} from './db/schema';
+import db from './db/connection';
+
+async function getCommunityLists(communityId: string) {
```

Comment:
Or: `getListsByCommunityId()` (to be in sync with other function names, e.g. `getCommunityBySlug()`)?